webview/__init__.py

```diff
@@ -41,9 +41,17 @@ def _initialize_imports():
 
     if not _initialized:
         if platform.system() == "Darwin":
-            import webview.cocoa as gui
+            try:
+                import webview.cocoa as gui
+            except ImportError:
+                try:
```

Comment:
Maybe change it to the import_error flag to be inline with the rest of the function.